worlds/stardew_valley/docs/setup_en.md

```diff
@@ -13,88 +21,27 @@
   can only be used with later releases of the world generator, that are not hosted on archipelago.gg yet.
 
 ## Optional Software
-- Archipelago from the [Archipelago Releases Page](https://github.com/ArchipelagoMW/Archipelago/releases)
-    * (Only for the TextClient)
-- Other Stardew Valley Mods [Nexus Mods](https://www.nexusmods.com/stardewvalley)
-    * There are [supported mods](https://github.com/agilbert1412/StardewArchipelago/blob/5.x.x/Documentation/Supported%20Mods.md) 
-  that you can add to your yaml to include them with the Archipelago randomization
+- I highly recommend Notepad, Winrar and Clippy

```

Comment:
```suggestion
- I highly recommend Wordpad, Winrar and Clippy

```
Notepad is unnecessary for this task, just use Wordpad.

Reply:
Something something C-x M-c M-butterfly
https://xkcd.com/378/